CHANGELOG.md

```diff
@@ -12,6 +12,9 @@ to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 * `jj rebase --skip-empty` has been renamed to `jj rebase --skip-emptied`
 
+* `jj backout --revision` has been renamed to `jj backout --revisions`.
+  The short alias `-r` is still supported.
```

Comment:
nit: might be better to leave flag name unchanged or aliased to singular form.

(This applies to various commands, and can be addressed later.)

Reply:
Right, I changed this for rebase as well. I remember there were at least some other commands which used `--revisions`. It seemed kind of inconsistent for now, which is why I changed it.

Reply:
I will merge this first. Perhaps we can change the flag name later.